networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/network/NetworkConfig.java

```diff
@@ -31,9 +32,22 @@
   private final boolean isDiscoveryEnabled;
   private final List<String> bootnodes;
   private final TargetPeerRange targetPeerRange;
-  private final boolean logWireCipher;
-  private final boolean logWirePlain;
-  private final boolean logMuxFrames;
+
+  // Gossip options
+  // https://github.com/ethereum/eth2.0-specs/blob/v0.11.1/specs/phase0/p2p-interface.md#the-gossip-domain-gossipsub
+  private int gossipD = 6;
+  private int gossipDLow = 4;
+  private int gossipDHigh = 12;
+  private int gossipDLazy = 6;
+  private Duration gossipFanoutTTL = Duration.ofSeconds(60);
+  private int gossipAdvertise = 3;
+  private int gossipHistory = 5;
+  private Duration gossipHeartbeatInterval = Duration.ofSeconds(1);
```

Comment:
Given these come from the spec I'd be tempted to just make them constants.  Unless there's some value in making them configurable?

I kind of like having them still come from the `NetworkConfig` object though.

Reply:
> I kind of like having them still come from the NetworkConfig object though.

@ajsutton not sure what do you mean exactly here, please check if the latest update looks better

Reply:
Yep that change is exactly what I meant. :)